wof/wofpy_config.py

```diff
@@ -16,62 +16,82 @@
 
 Usage:
     wofpy_config INDIR
-    wofpy_config INDIR [--mode=<development,production>]
+    wofpy_config INDIR [--mode=<development,production>] [--overwrite=<soft,hard,none>]
 
-    wofpy_config (-h | --help | -v | --version)
+    wofpy_config (-h | --help | -v | --version | -m | --mode | -o | --overwrite)
 
 Examples:
     wofpy_config wofpyserver
+    wofpy_config wofpyserver --mode=production --overwrite=soft
 
 Arguments:
-  directory     Configuration directory.
+  directory               Configuration directory.
 
 Options:
-  -h --help     Show this screen.
-  -v --version  Show version.
-  --mode=development   deploy mode [default: development]
+  -h --help               Show this screen.
+  -v --version            Show version.
+  -m --mode=development   Deploympent mode [default: development].
+  -o --overwrite=none     Overwrite everything *hard*,
+                          or only adds extra files to an exsting directory *soft*
+                          [default: none].
 """
 
 _ROOT = os.path.abspath(os.path.dirname(wof.__file__))
 _CONFIG = os.path.join(_ROOT, 'examples', 'production_configs')
 _ODM2_TIMESERIES = os.path.join(_ROOT, 'examples', 'flask', 'odm2', 'timeseries')
 
 
-def makedirs(directory):
+def makedirs(directory, overwrite=None):
     if not os.path.exists(directory):
         os.makedirs(directory)
+    elif os.path.exists(directory) and overwrite in ['soft', 'hard']:
+        if overwrite == 'hard':
+            shutil.rmtree(directory)
```

Comment:
I mean that you cannot call it programatically when there is a user expected input.
Either way I am not convienced that is a good idea...